blog/12.15.0.ga.adoc

```diff
@@ -0,0 +1,537 @@
+
+== What is new?
+
+Our main focus for this release was a new tooling for the Quarkus framework, improvements for container based development and bug fixing.
+Eclipse 2020-03 itself has a lot of new cool stuff but let me highlight just a few updates in both Eclipse 2020-03 and JBoss Tools plugins that I think are worth mentioning.
+
+=== OpenShift
+
+==== OpenShift Container Platform 4.4 support
+
+With the new OpenShift Container Platform (OCP) 4.4 now available, JBoss Tools
+is compatible with this major release in a transparent way. Just define your
+connection to your OCP 4.4 based cluster as you did before for an OCP 3 cluster,
+and use the tooling !
+
+=== Quarkus
+
+==== Language support for Kubernetes, Openshift, S2i and Docker properties
+
+There is now completion, hover, documentation and validation for kubernetes.*,
+openshift.*, s2i.*, docker.* properties
+
+image::/documentation/whatsnew/quarkus/images/quarkus20.png[]
+
+Enter `kubernetes` prefix:
+
+image::/documentation/whatsnew/quarkus/images/quarkus21.png[]
+
+Enter `openshift` prefix:
+
+image::/documentation/whatsnew/quarkus/images/quarkus22.png[]
+
+Enter `s2i` prefix:
+
+image::/documentation/whatsnew/quarkus/images/quarkus23.png[]
+
+== Language support for MicroProfile REST Client properties
+
+Likewise, there is now completion, hover, documentation and validation for the
+MicroProfile properties from REST Client.
+
+After registering a REST client using `@RegisterRestClient` like so:
+[source,java]
+----
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient
+public interface MyServiceClient {
+	@GET
+    @Path("/greet")
+    Response greet();
+}
+----
+
+The related MicroProfile Rest config properties will have language feature support
+(completion, hover, validation, etc.).
+
+image::/documentation/whatsnew/quarkus/images/quarkus24.png[]
+
+Change the Java code so that the configuration key is changed:
+[source,java]
+----
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient(configKey = "myclient")
+public interface MyServiceClient {
+	@GET
+    @Path("/greet")
+    Response greet();
+}
+----
+
+and notice the code assist is changed accordingly:
+
+image::/documentation/whatsnew/quarkus/images/quarkus25.png[]
+
+== Language support for MicroProfile Health
+
+Likewise, there is now completion, hover, documentation and validation for the
+MicroProfile Health artifacts.
+
+So if you have the following Health class:
+
+[source,java]
+----
+package org.acme;
+
+import org.eclipse.microprofile.health.Health;
+
+@Health
+public class MyHealth {
+
+}
+----
+
+you will get a validation error (as the class does not implement the `HealthCheck`
```

Comment:
there is a missing parenthesis ) .

Reply:
and also, the screenshot shows a validation warning, not an error, maybe worth change the test to 'a validation message' ?